config/settings.yml

```diff
@@ -1383,6 +1383,7 @@
       :performance_rollup_purging_start_delay: 5.minutes
       :policy_events_purge_interval: 1.day
       :poll: 15.seconds
+      :report_result_purge_interval: 1.months
```

Comment:
The thing I don't like about 1.month is that if the server is restarted, the timer also restarts...not sure how often appliances are restarted.